app_flutter/test/utils/fake_flutter_build.dart

```diff
@@ -10,6 +10,7 @@ import 'package:cocoon_service/protos.dart' show Commit, CommitStatus, Task;
 
 import 'package:app_flutter/service/google_authentication.dart';
 import 'package:app_flutter/state/flutter_build.dart';
+import 'package:app_flutter/state/brooks.dart';
```

Comment:
nit: This should be above the previous import

Reply:
fixed